### ISSUE TITLE
qt-gui: keys with /

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/libelektra#173]
keys should be created by using keyAddBaseName that properly escapes /

There should be a way to create subkeys to keys that were created before (currently not possible because key is only at right view)

**Comment by 0003088** [comment]
Slash escape fixed in 8bba76d
Subkey issue fixed in dd8632d

**Comment by markus2330** [comment]
I fixed the undefined behaviour because of missing KEY_END in 10af7828755b0d971a0494e3651f7dde68912a05.

Otherwise nicely done!

Only the ordering of the Key Names is sometimes a bit confusing. (Lets discuss that on Tuesday)


```
#0  0x00007fb4d322302e in keySetOwner (key=key@entry=0x7fb4b40368f0, newOwner=newOwner@entry=0x4 <Address 0x4 out of bounds>) at /home/markus/Projekte/Elektra/current/src/libelektra/keyname.c:1262
#1  0x00007fb4d3226a44 in keyVInit (key=key@entry=0x7fb4b40368f0, name=name@entry=0x7fb4c7fa5928 "user/guitest/b/x", va=va@entry=0x7fff0a244508) at /home/markus/Projekte/Elektra/current/src/libelektra/keyhelpers.c:426
#2  0x00007fb4d3226fe6 in keyVNew (name=0x7fb4c7fa5928 "user/guitest/b/x", va=0x7fff0a244508) at /home/markus/Projekte/Elektra/current/src/libelektra/key.c:223
#3  0x000000000041bc81 in kdb::Key::Key (this=0x7fff0a2446a0, keyName=...) at /home/markus/Projekte/Elektra/current/src/bindings/cpp/include/key.hpp:548
#4  0x000000000043ad8c in NewKeyCommand::NewKeyCommand (this=0x7fb4b403d010, parentNode=..., data=<optimized out>, isBelow=false, parent=<optimized out>) at /home/markus/Projekte/Elektra/current/src/tools/qt-gui/src/newkeycommand.cpp:18
#5  0x000000000042d955 in UndoManager::createNewKeyCommand (this=0x7fff0a2477e0, model=<optimized out>, idx=<optimized out>, data=0x1adec30, isBelow=false) at /home/markus/Projekte/Elektra/current/src/tools/qt-gui/src/undomanager.cpp:45
```
